tests/e2e/gpu-feature-discovery_test.go

```diff
@@ -63,6 +64,15 @@ var _ = NVDescribe("GPU Feature Discovery", func() {
 		"nvidia.com/mps.capable":          "[true|false]",
 	}
 
+	defaultCollectorObjects := []string{
+		"pods",
+		"nodes",
+		"namespaces",
+		"deployments",
+		"demonsets",
+		"nodefeatures",
```

Comment:
Is the reason for a different default just because we don't launch a job here? Is there harm in also defining jobs here?

Reply:
no harm, if no jobs, it will simply do nothing because it won't find any jobs objects.

My idea is to make this modular so it can be used for any component e2e test. as discussed previously, I am already thinking this `DiagnosticsCollector` should live at `k8s-test-infra` so it can be imported by all the projects. so it might seem irrelevant here, I agree, but moving forward the `DiagnosticCollector` could also have `ServiceAccount` `role` and many other k8s objects we consider important during  debugging

Reply:
no harm, if no jobs, it will simply do nothing because it won't find any jobs objects.

My idea is to make this modular so it can be used for any component e2e test. as discussed previously, I am already thinking this `DiagnosticsCollector` should live at `k8s-test-infra` so it can be imported by all the projects. so it might seem irrelevant here, I agree, but moving forward the `DiagnosticCollector` could also have `ServiceAccount` `role` and many other k8s objects we consider important during  debugging